src/service/video/autoplay.js

```diff
@@ -253,15 +252,15 @@ export class AutoplayEntry {
   }
 
   /**
-   * @param {!../../observable.Observable<boolean>} playbackObservable
+   * Delegates autoplay so that it's triggered by a different module.
    * @public
    */
-  delegateTo(playbackObservable) {
+  delegate() {
     if (this.unlistener_) {
       this.unlistener_();
     }
-    this.unlistener_ =
-        playbackObservable.add(isPlaying => this.trigger_(isPlaying));
+    this.video.pause();
```

Comment:
Is this right?  Isn't it possible that `amp-story-page` has already called `play()` and this will pause it?

Reply:
Yup! This will not be called if the signal was already received.

(Note that `video/autoplay.js` is under experimental use by flag `video-service`).